### PR TITLE
Replaced FlatButton to TextButton

### PR DESCRIPTION
**Pull request by Ravig786**
Fixed Deprecated Code for example changed FlatButton to TextButton.

### CODE REVIEW

**Comment by Sagar2366** [comment]
@Ravig786 create issue for this PR and update PR description with all details. Follow these templated for issue and PR - https://github.com/PalisadoesFoundation/talawa/tree/master/templates

**Comment by Ravig786** [comment]
> @Ravig786 create issue for this PR and update PR description with all details. Follow these templated for issue and PR - https://github.com/PalisadoesFoundation/talawa/tree/master/templates

What kind of change does this PR introduce: It removes some of the deprecated code.
Issue Number: #374

**Comment by Sagar2366** [comment]
Seems like this is duplicate issue - check https://github.com/PalisadoesFoundation/talawa/pull/214

**Comment by Sagar2366** [comment]
@Ravig786 resolve conflicts on this PR. Disable auto-formatter before sending PR for review.
Remove changes which are not required to fix this issue.

**Comment by Sagar2366** [comment]
@Ravig786  few problems with this PR-
You have not yet updated PR as per PR template.
Adding extra code which is not required to fix this issue (disable code auto-formatter before pushing changes to PR)
In issue number - you are actually providing link for this same PR.

**Comment by Sagar2366** [comment]
@Ravig786 please open new PR, with only changes to replace flatbutton to textbutton.

File: lib/views/pages/organization/remove_member.dart [hunk 1]
```diff
@@ -0,0 +1,175 @@
+//flutter imported packages
```
Comment (Sagar2366) [inline lib/views/pages/organization/remove_member.dart:1]: Why are you adding this file as part of fixing this issue ?
Is it relevant ?

**Pull request closed without merging**